site/layouts/shortcodes/card.html

```diff
@@ -22,9 +22,11 @@
 {{- $rtl := .Get "rtl" | default false -}}
 
 <div class="card{{ if eq $borders false }} border-0{{ end }}{{ if and $borders (eq $background "dark") }} border-dark{{ end }}">
-  <svg class="bd-placeholder-img card-img-top" width="100%" height="162" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="{{ if $rtl }}تعليق على الصورة{{ else }}Image cap{{ end }}" preserveAspectRatio="xMidYMid slice" focusable="false">
+  <svg class="bd-placeholder-img card-img-top" width="100%" height="169" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="{{ if $rtl }}تعليق على الصورة{{ else }}Image cap{{ end }}" preserveAspectRatio="xMidYMid slice" focusable="false">
```

Comment:
Currently wondering if `Image cap` is a good value for an `aria-label` attribute?
It's outside the frame of the current PR so we may discuss it elsewhere.

Reply:
Seems to be on Bootstrap Side so yes, let's tackle it there directly!